src/ducks/Alert/components/AlertModalContent/StepsContent/MetricAndConditions/ConditionsSelector/OperationSelector/OperationSelector.js

```diff
@@ -45,9 +46,10 @@ const OperationSelector = ({ metric, isWallet }) => {
     }
   }, [operation, count])
 
-  const hasPriceIcon = metric.category === 'Financial' || metric.key === 'price_usd'
-  const isPositiveMetric = metric.key !== 'exchange_balance'
-  const isPercentIcon = PERCENT_OPERATIONS.includes(operation.value)
+  const hasPriceIcon = USD_METRICS.includes(metric.key)
+  const isPositiveMetric = !NEGATIVE_METRICS.includes(metric.key)
+  const isPercentIcon =
+    PERCENT_OPERATIONS.includes(operation.value) || PERCENT_METRICS.includes(metric.key)
```

Comment:
`.includes` is the `O(n)` lookup. We can make it `O(1)` by changing `USD_METRICS` (and other constants) type to `Set` and using `USD_METRICS.has(metric.key)` method